src/main/java/apprenticeship/Board.java

```diff
@@ -0,0 +1,28 @@
+package apprenticeship;
+
+import java.util.Arrays;
+
+public class Board {
+    public char cells[];
```

Comment:
cells[] has to be public because I call tictactoe.cells various times in the test.
when I make it private, it doesn't let me.